backend/src/routing/routes/auth.ts

```diff
@@ -0,0 +1,44 @@
+import {Router} from 'express';
+import {generators} from 'openid-client';
+import {redirectLink, iservConnectToLoginError} from '../../auth/staticAuthStrings';
+
+const code_verifier = generators.codeVerifier();
+
+export default (): Router => {
+    const router = Router();
+
+    router.get('/login', async (req, res, next) => {
```

Comment:
Z. 10 - 28 und Z. 30 - 41 sollte in einen Controller ausgelagert werden. Bei Fragen dazu bitte an Benito wenden :D